applications/luci-app-usteer/htdocs/luci-static/resources/view/usteer/usteer.js

```diff
@@ -91,25 +91,55 @@ function collectWlanAPInfoEntries(connectioninfo_table_entries, wlanAPInfos) {
 	}
 };
 
+function tootltip(mac, IP, hostname) {
+	var body= E([]);
+	var i=0;
+	body.appendChild(E('div', mac));
+	if (typeof IP !== 'undefined') {
+		while ((typeof IP['ipaddrs'][i] !== 'undefined') && (String(IP['ipaddrs'][i]).length > 0)) {
+			body.appendChild(E('div', IP['ipaddrs'][i]));
+			i++;
+		}
+		
+		i=0;
+		while ((typeof IP['ip6addrs'][i] !== 'undefined') && (String(IP['ip6addrs'][i]).length > 0)) {
```

Comment:
using `&& String...` might behave unexpectedly: what is the truth test here? That you coerced something to a string? Or that length is > 0? Perhaps skip String()

Reply:
I tested for length >0 as I encountered a empty string as only entry in the past. With the change above this is now not needed anymore.